src/main/java/gift/ProductWebController.java

```diff
@@ -29,11 +28,13 @@ public String getProductsPage(Model model) {
     }
 
     @PostMapping(consumes = "application/x-www-form-urlencoded;charset=UTF-8")
-    public String postProduct(@RequestParam String name, @RequestParam BigDecimal price, @RequestParam String imageUrl) {
-        Product product = new Product();
-        product.setName(name);
-        product.setPrice(price);
-        product.setImageUrl(imageUrl);
+    public String postProduct(@RequestParam String name, @RequestParam BigDecimal price, @RequestParam String imageUrl, @RequestParam String description) {
```

Comment:
원모님은 가로 제한을 몇으로 두고 개발하시나요?
줄이 길어지면 줄바꿈을 활용해서 가독성을 높여볼 수 있을 것 같아요

```suggestion
    public String postProduct(@RequestParam String name,
                              @RequestParam BigDecimal price,
                              @RequestParam String imageUrl,
                              @RequestParam String description) {
```